tests/rptest/scale_tests/node_operations_fuzzy_test.py

```diff
@@ -89,6 +95,7 @@ def _create_random_topics(self, count, compacted):
             compacted_topics=[True, False])
     def test_node_operations(self, enable_failures, num_to_upgrade,
                              compacted_topics):
+        self.admin_fuzz = None
```

Comment:
nit: move to constructor (if we add a test which doesn't use / initialize admin_fuzz - `tearDown` will fail)